bin/pd/Makefile

```diff
@@ -183,6 +184,30 @@ $(workdir)/briefingt.csv:
 $(workdir)/filtered/briefingt.csv: $(workdir)/briefingt.csv
 	$(fdir)/filter_briefingt.py < $< > $@
 
+###
+###  Question Period Notes
+###
+.PHONY: qpnotes
+qpnotes: upload-qpnotes rebuild-qpnotes
+
+.PHONY: upload-qpnotes
+upload-qpnotes: $(workdir)/filtered/qpnotes.csv
+	@echo Uploading Question Period Notes...
+	@$(ckanapi) action resource_patch -c $(registry_ini) \
+	id=ba1284e5-db5f-43e1-854f-2beb7fe8f04b upload@"$(workdir)/filtered/qpnotes.csv"
```

Comment:
Where did you get the resource ID from here?

Reply:
`paster recombinant update -a -c /opt/tbs/ckan/conf/development.ini` gave it to me

Reply:
no that's not right. The resources here are resources in datasets on the registry that are used to publish the csv files. Usually we get the business team to create these for us.

Reply:
need to fix this before merge to master (staging doesn't matter)